app/views/posts/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<table BORDER>
+  <thead>
+    <tr> 
+      <th>TITLE</th><th>DESCRIPTION</th><th>USER_ID</th><th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><%= @post.title %></td>
+      <td><%= @post.description %></td>
+      <td><%= @post.user_id %></td>
+      <td>
+        <%= link_to "Edit post", edit_post_path(@post)%> 
+        <%= link_to 'Delete', post_path(@post), method: :delete, data: {confirm: "Are you sure?"} %>
```

Comment:
In that approach you are allowing all users (even not logged in) to edit/delete post, if you want to add those fields you should do it conditionally checking if current_user is the author of post.